src/tests/Dashboard/action/dictionaryAction.test.js

```diff
@@ -381,6 +382,49 @@ describe('Test suite for dictionary actions', () => {
       expect(store.getActions()).toEqual(expectedActions);
     });
   });
+
+  it('should retire a concept when the retire button is clicked', () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        response: sampleRetiredConcept,
+      });
+    });
+    const store = mockStore({ payload: {} });
+    return store.dispatch(retireConcept(sampleConcept.url, { retired: true })).then((result) => {
+      expect(result.retired).toEqual(true);
+    });
+  });
+
+  it('should unretire a concept when the unretire button is clicked', () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        response: sampleConcept,
+      });
+    });
+    const store = mockStore({ payload: {} });
```

Comment:
Same as above

Reply:
`DictionaryConcepts.test.jsx` lines 528 - 621

Reply:
Same as above

Reply:
I have changed the title to `should unretire a concept when the retireConcept action is triggered with the false argument`